packages/framework-provider-azure-infrastructure/test/infrastructure/functions/event-handler-functions.test.ts

```diff
@@ -20,7 +20,7 @@ describe('Creating event-handler-functions', () => {
     expect(definition.config.bindings[0].databaseName).to.be.equal('new-booster-app-app')
     expect(definition.config.bindings[0].collectionName).to.be.equal('new-booster-app-app-events-store')
     expect(definition.config.bindings[0].createLeaseCollectionIfNotExists).to.be.equal('true')
-    expect(definition.config.scriptFile).to.be.equal('../dist/index.js')
+    expect(definition.config.scriptFile).to.be.oneOf(['../dist/index.js', '..\\dist\\index.js'])
```

Comment:
I think we should use path.join instead, for consistency, since we need to use this later (e.g. 7b6e234b3dcad4987c8cdc1df82080abca5d3630). Let me know if you agree.

Reply:
true, made the change as well as changing 'set' to 'cross-env' and removing the ampersand, seems to still pass the tests, waiting on aws infrastructure before making a new pr